File: sem4/MPP/proiect/appClientServer/CompetitionNetworking/src/main/java/org/example/dto/TaskDTO.java
package org.example.dto;

import org.example.model.Task;

import java.io.Serializable;

public class TaskDTO extends Task implements Serializable {
    private int nrChildren;
    public TaskDTO(String description, int minAgeCat, int maxAgeCat, int number) {
        super(description, minAgeCat, maxAgeCat);
        this.nrChildren = number;
    }

    public int getNrChildren() {
        return nrChildren;
    }

    public void setNrChildren(int number) {
        this.nrChildren = number;
    }
}

File: sem4/MPP/proiectC#/AppClientServer/Networking/TaskDTO.cs
using System;

namespace Networking
{
    [Serializable]
    public class TaskDTO
    {
        public string description { get; set; }
        public int ageCatStart { get; set; }
        public int ageCatEnd { get; set; }
        public int nrChildren { get; set; }

        public TaskDTO(string description, int ageCatStart, int ageCatEnd, int nrKids)
        {
            this.description = description;
            this.ageCatStart = ageCatStart;
            this.ageCatEnd = ageCatEnd;
            this.nrChildren = nrKids;
        }

        public virtual string Description
        {
            get
            {
                return description;
            }
        }
        
        public virtual int AgeCatStart
        {
            get
            {
                return ageCatStart;
            }
        }
        public virtual int AgeCatEnd
        {
            get
            {
                return ageCatEnd;
            }
        }
        public virtual int NrChildren
        {
            get
            {
                return nrChildren;
            }
        }
    }
}